app/src/main/res/values/strings.xml

```diff
@@ -154,6 +154,7 @@
     <string name="ownership_resend_code_by_sms">Time left until code sent via SMS expires: %1$d</string>
     <string name="resend_code">Resend code</string>
     <string name="sms_code_expired">The SMS code has expired. Please try again.</string>
+    <string name="copy_code_telegram">Copy the code sent to you via Telegram</string>
```

Comment:
Please change text to:

"In case of Telegram, please enter the code without copying it from Telegram. Copying the code is a security risk and Telegram may take some actions to protect your account."